packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -258,18 +259,18 @@ export default class ConfigParser {
         const multiRun = this._config.multiRun
         // when CLI --spec is explicitly specified, this._config.specs contains the filtered
         // specs matching the passed pattern else the specs defined inside the config are returned
-        let specs = ConfigParser.getFilePaths(this._config.specs!, this._config.rootDir, this._pathService)
-        let exclude = ConfigParser.getFilePaths(this._config.exclude!, this._config.rootDir, this._pathService)
+        let specs = this.findSpecs(this._config.specs!)
```

Comment:
Can we just amend the `ConfigParser.getFilePaths` instead of introducing a new function?